docs/resource-editing.md

```diff
@@ -1,71 +1,78 @@
 # Resource Editing
 
-Monokle allows you to edit resource manifests either in a Source Editor
+Monokle allows you to edit resource manifests either in a Source Editor or a Form-based Editor which hides the underlying YAML complexity.
 
 ## Source Editor
 
-Selecting either a file or resource will show its contents in the source editor to the right. The editor will syntax-highlight for
-yaml and provide context-sensitive help and auto-complete functionality for standard kubernetes objects.
+Selecting either a file or resource will show its contents in the Source Editor to the right. The editor will syntax-highlight for
+YAML and provide context-sensitive help and auto-complete functionality for standard Kubernetes objects.
 
-Right-clicking in the editor shows a list of available commands - and pressing F1 opens its command-palette:
+Right-clicking in the editor shows a list of available commands. Pressing F1 opens its command-palette:
 
 ![Source Editor Command Palette](img/source-editor-command-palette.png)
 
 ### Editing Resources
 
-When editing resources (not files) the editor is schema-aware for all native Kubernetes resources, which provides
-auto-complete (Ctrl-Space) and context-sensitive hover documentation
+When editing resources (not files), the editor is schema-aware for all native Kubernetes resources, which provides
+auto-complete (Ctrl-Space) and context-sensitive hover documentation.
 
 ![Source Editor Auto Complete](img/source-editor-auto-complete.png)
 
 ![Source Editor Context Hover](img/source-editor-context-hover.png)
 
-#### Resource Links
+### Resource Links
 
-Resource links are marked in the gutter and underlined in the editor, with corresponding hover/popup windows to show linked resources:
+Resource links are marked in the gutter and underlined in the Editor, with corresponding hover/popup windows to show linked resources:
 
 ![Source Editor Resource Links](img/source-editor-resource-links.png)
 
-Broken links are shown with yellow triangles as in the Navigator:
+Broken links are shown with yellow triangles in the Editor as in the Navigator:
 
 ![Source Editor Broken Links](img/source-editor-broken-links.png)
-
+  
 ## Add Resource
 
 While using Monokle, you can directly add new K8s resources. Once you have browsed and added your project folder, click on the “New Resource” button at the top of the navigator to launch the “Add New Resource” dialog.
 
-![Add Resource](img/add-resource.png)
+![Add Resource](img/add-resource-1.5.0.png)
```

Comment:
should we have a 1.5.0 screenshot here?

Reply:
These screenshots will be that same for 1.4.0 and 1.5.0, so I renamed them 1.5.0 to note that these images will not need to be updated for the next release.  I will verify with the final release build to make sure, though.

Reply:
There are only a few of these.  I can come up with another process to keep up with this if it does not make sense.

Reply:
it makes sense, thanks for clarifying